src/http/server.ts

```diff
@@ -426,65 +475,41 @@ export class Server {
    *     Returns the response as stringified JSON. This is only used for unit
    *     testing purposes.
    */
-  public handleHttpRequestForStaticPathAsset(request: any): any {
+  public async handleHttpRequestForStaticPathAsset(
+    request: Drash.Http.Request,
+  ): Promise<Drash.Interfaces.ResponseOutput> {
```

Comment:
Has the doc block been updated to reflect this?

Reply:
updated